elements/pfe-accordion/src/pfe-accordion-header.scss

```diff
@@ -63,3 +64,39 @@ button[aria-expanded="true"] {
     @include pfe-accordion-variables($state: expanded, $theme: #{$context});
   }
 }
+
+/// ===========================================================================
+/// DISCLOSURE STYLES
+/// ===========================================================================
+
+:host([pfe-disclosure="true"]) {
```

Comment:
@chrisdo1 I'm saying this blindly without knowing how difficult it might be to fix it, but we are duplicating a lot of styles that maybe don't need to be printed twice?
https://www.evernote.com/l/AA6a85jK0W5J1LGziQ1c6FIwI4CR56BofyU
If nothing else, let's flag this as a @todo to tidy later, but maybe see if we could just override the CSS vars instead of calling the mixin again?

Reply:
@starryeyez024 good catch! I removed the `pfe-trigger` mixin and re-added the padding properties. Maybe it doesn't make sense to have the left/right arguments in this mixin since we're not currently using `$align: right` anywhere but here.